tests/test_cases/test_cocotb/test_cocotb_35.py

```diff
@@ -0,0 +1,109 @@
+"""
+This file contains tests that use python 3.5-only syntax.
```

Comment:
nit: Python 3.5+ syntax (the "only" initially got me confused)

Reply:
how about "syntax introduced in python 3.5"?

Reply:
sure